README.md

```diff
@@ -1,95 +1,78 @@
-Hardware
-========
+# Pixhawk Hardware Designs
 
-[![Join the chat at https://gitter.im/PX4/Hardware](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PX4/Hardware?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Join the chat at https://gitter.im/PX4/Hardware](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PX4/Hardware?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
```

Comment:
@LorenzMeier Do we monitor these channels? Is gitter really where you want to have discussions/feedback?

The next iteration should add info about communicating with project, contributing to designs, etc. Will need your help on that.

Reply:
Or is channel https://discuss.px4.io/c/pixhawk ? As discussed, IMO this has been used for other purposes, so if we wish to have a channel for standards discussion it probably should be called Pixhawk Standards or similar.

Reply:
OK, so new proposal is move gitter to Pixhawk/Hardware and use that as only discussion channel. Then add docs on dev call in this update, and point to those docs too.